src/styles/help.scss

```diff
@@ -0,0 +1,10 @@
+button.help {
```

Comment:
Decide if we want these separate files. Currently `layout.scss` contains some non-layout stuff around components (including help) but I've keep here to avoid confusing merge conflicts.